draft-ietf-idr-link-bandwidth.xml

```diff
@@ -259,13 +259,35 @@
       the above preference.</t>
     </section>
 
+    <section anchor="History" title="Document History">
+      <t>The BGP Link Bandwidth Extended Community has evolved over several
+      versions of the IETF draft. In the earlier versions up to
+      draft-ietf-idr-link-bandwidth-08, only the non-transitive version of the
+      link bandwidth extended community was supported. However, starting from
+      draft-ietf-idr-link-bandwidth-09, both transitive and non-transitive
+      versions of the link bandwidth extended community are supported.</t>
+
+      <t>An old sender/receiver is a BGP speakers either use procedures upto
+      draft
+      (https://datatracker.ietf.org/doc/html/draft-ietf-idr-link-bandwidth-08)
+      or undocumented behavior for link bandwidth extended community.</t>
+
+      <t>A new sender/receiver is a BGP speaker that implements procedures
+      specified in this document. </t>
+
+      <t>Old receivers will not be able to interpret or utilize the new
```

Comment:
Old receives may not be able to ..

Reply:
ok, u mention 'new procedures'. if u remove the new, then it is a 'may'.

Reply:
I have made this change. Please review.